aptos-move/framework/aptos-token/sources/token.move

```diff
@@ -1382,6 +1382,60 @@ module aptos_token::token {
         token_data.royalty
     }
 
+    public fun get_tokendata_id(token_id: TokenId): TokenDataId {
```

Comment:
Can you add docs for all the getters here?

Reply:
added some. for the rest, we can add later in a separate diff since doc updating doesn't require a mainnet release.